ansible/roles/vm_set/templates/cleanup.sh.j2

```diff
@@ -17,6 +17,16 @@ rm -f {{ root_path }}/disks/*
 # remove all ovs bridges
 test -z "$(ovs-vsctl list-br)" || ovs-vsctl list-br | xargs -I % ovs-vsctl del-br %
 
+# stop exabgp process before stopping ptf container
+test -z "$(docker ps --format "table {{.Names}}" | grep ptf)"
+ptf_container_name=$(docker ps --format "table {{.Names}}" | grep ptf)
+for ptf in  $ptf_container_name;
+do
+  test -z "$(docker exec $ptf supervisorctl status | grep exabgp | awk '{print $1}')"
+  exabgp_process_name="$(docker exec $ptf supervisorctl status | grep exabgp | awk '{print $1}')"
+  docker exec $ptf supervisorctl stop $exabgp_process_name
```

Comment:
There could be multiple exabgp services running in PTF container.
I am not sure if that scenario can be handled by the code here.

Reply:
supervisorctl stop can stop multiple processes, so multiple exabgp services could be stopped